extensions/wikia/PortableInfobox/styles/PortableInfobox.scss

```diff
@@ -150,7 +151,31 @@
 
 // **********  Collapsible groups  ********** //
 
+.pi-collapse {
+	.pi-header {
+		@include right-chevron($infobox-chevron-size, 2px, 'up', $color-text);
+		padding-right: $infobox-item-margin * 3 + $infobox-chevron-size;
+		position: relative;
+
+		&:after {
+			display: block;
+			margin-top: -($infobox-chevron-size / 2) + 2;
+			position: absolute;
+			right: $infobox-item-margin * 2;
+			top: 50%;
+		}
+	}
+}
+
 .pi-collapse-closed {
+	border-bottom: none;
+
+	.pi-header::after {
+		margin-top: -($infobox-chevron-size / 2);
+		-webkit-transform: rotate(135deg);
```

Comment:
I know that properties here are supposed to by ordered alphabetically - so I think "-..." comes first - but not sure - maybe check other examples?